web/src/app/selection/QuerySelect.js

```diff
@@ -61,6 +63,7 @@ function makeUseValues(query, mapNode) {
       pause: !value.length,
       variables,
       requestPolicy: 'cache-first',
+      context: noSuspenseContext,
```

Comment:
These 2 will also need to be behind a memo

Reply:
Not actually needed in this case, otherwise, it'd render in an infinite loop.

Reply:
I'm basing this off of their documentation, where when using context to pair that with useMEmo, and having talked to their devs when asking for help:

https://github.com/urql-graphql/urql/discussions/3421#discussioncomment-7430479

Reply:
Adding context/ref. from offline convo:

Under the covers, this is basically the "issue" on why it needs stable references:
```js
console.log({} === {}) // false
```

Each time `useQuery` gets a new context object (even if it has the exact same props) it will re-fetch detecting a change. `useMemo` is a way to return a _new_ object, ONLY if a dependency changes.

There's logic that's basically if `if (newContext !== oldContext) refetch()`. So if you inline `{ suspense: false }` it's actually creating a new object reference every time the function runs, and it will **always** appear to be changing without something like `useMemo` (and creating an infinite render loop).

In our case, since we just need to set an option with no dependencies or changes, we can skip that step by just declaring a const at the module level. That will never change, and so we don't get the render issue.

Reply:
Thank you for sharing here! I'll resolve the comments.